packages/runtime/src/editor-preview.main.ts

```diff
@@ -841,6 +841,12 @@ export const createRoot = (
               }
               break
           }
+          if (message.data.canceled) {
```

Comment:
This is only conditionally on cancel (ESC during drag) to make the drop feel instant instead of waiting for the DOM update.